documents/order.md

```diff
@@ -275,6 +275,75 @@ Razorpay::Order.edit(orderId,para_attr)
 -------------------------------------------------------------------------------------------------------
 
 
+### View RTO/Risk Reasons
+
+```go
+orderId = "order_XXXXXXXXXXXXX1"
+
+Razorpay::Order.fetch(orderId).view_rto
```

Comment:
please remove the method chaining.

Reply:
Updated doc

https://github.com/razorpay/razorpay-ruby/blob/c06d03c69ec3eeb816196e5a0ce5b663929c84f5/documents/order.md?plain=1#L283